### PR TITLE
samples: openthread: enable PSA crypto API in non-secure builds

### PR DESCRIPTION
**Pull request by Damian-Nordic**
We've recently added support for OpenThread crypto backend based on PSA crypto API to Zephyr. Let's make use of it in
our OpenThread samples.

Signed-off-by: Damian Krolik <damian.krolik@nordicsemi.no>

### CODE REVIEW

**Review by LuDuda — APPROVED** [review]

**Review by LuDuda — COMMENTED** [review]
Just spotted that we don't have information about this feature in change-log. Could we add such, that OT can be now built with TF-M...?

**Review by rlubos — APPROVED** [review]
The PR likely needs a rebase according to NCS CI channel.

**Review by lmaciejonczyk — APPROVED** [review]

**Review by MarekPorwisz — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -192,6 +192,14 @@ nRF9160 samples
   * The sample has been removed.
     nRF Cloud A-GPS and P-GPS are demonstrated in the :ref:`gnss_sample` sample.
 
+OpenThread samples
+------------------
+
+* Added:
+
+  * Support for ``nrf5340dk_nrf5340_cpuapp_ns`` build target for :ref:`zephyr:nrf5340dk_nrf5340`.
+    This allows to build the OpenThread samples with Trusted Firmware-M and the usage of PSA crypto API.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:201]:
```suggestion
    This allows to build the OpenThread samples with Trusted Firmware-M and the PSA crypto API support.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
I don't like how "build with ... the usage" sounds. Hence the edit.

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (Damian-Nordic) [reply doc/nrf/releases/release-notes-changelog.rst]: Done.

**Review by greg-fer — APPROVED** [review]

**Pull request merged**